resources/scponly_user.rb

```diff
@@ -0,0 +1,124 @@
+# true To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+
+resource_name :scponly_user
+
+default_action :create
+
+property :write_dir, String, default: 'incoming'
+property :public_key, String
+property :private_key, String
+property :chroot, [true, false], default: true
+property :altroot, String
```

Comment:
I really prefer this to be set here by default. In addition, I'm thinking this might be a better location in the long term along with defaulting to creating new chroots for each resource until otherwise specified.
```suggestion
property :altroot, String, "/var/lib/chroots/#{new_resource.name}"
```